Не собирается проект

Добрый день!
Пытаюсь собрать проект, но выдаются ошибки. Лог работы скрипта приложил 
[BuildLog.docx](https://github.com/diadoc/diadocsdk-csharp/files/390011/BuildLog.docx)
Ругается на Cryptography\StreamCrypt.cs(123,60): error CS1056: Unexpected character '$' [D:\Projects\diadocsdk-csharp-master\src\DiadocApi-net35.csproj]

В чем может быть проблема?

--- a/Samples/Diadoc.Console/Diadoc.Console.csproj
+++ b/Samples/Diadoc.Console/Diadoc.Console.csproj
@@ -35,6 +35,9 @@
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="protobuf-net">
+      <HintPath>..\..\packages\protobuf-net.1.0.0.280\lib\protobuf-net.dll</HintPath>
+    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
@@ -92,7 +95,9 @@
       <Name>DiadocApi</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.

--- a/Samples/Diadoc.Console/packages.config
+++ b/Samples/Diadoc.Console/packages.config
@@ -1,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="protobuf-net" version="1.0.0.280" targetFramework="net40" />
+</packages>

--- a/Samples/Diadoc.Samples/Diadoc.Samples.csproj
+++ b/Samples/Diadoc.Samples/Diadoc.Samples.csproj
@@ -35,6 +35,9 @@
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="protobuf-net">
+      <HintPath>..\..\packages\protobuf-net.1.0.0.280\lib\protobuf-net.dll</HintPath>
+    </Reference>
     <Reference Include="System" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
@@ -69,7 +72,9 @@
       <Name>DiadocApi</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.

--- a/Samples/Diadoc.Samples/packages.config
+++ b/Samples/Diadoc.Samples/packages.config
@@ -1,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="protobuf-net" version="1.0.0.280" targetFramework="net40" />
+</packages>

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -1,4 +1,5 @@
 version: 1.0.{build}
+os: Visual Studio 2013
 assembly_info:
   patch: false
 platform: Any CPU

--- a/src/Cryptography/ExtendedWinApiCrypt.cs
+++ b/src/Cryptography/ExtendedWinApiCrypt.cs
@@ -24,23 +24,35 @@ namespace Diadoc.Api.Cryptography
 			streamCrypt = new StreamCrypt(new X509Certificate2(certificate));
 		}
 
-		public byte[] Sign(byte[] content) =>
-			streamCrypt.Encode(content);
+		public byte[] Sign(byte[] content)
+		{
+			return streamCrypt.Encode(content);
+		}
 
-		public byte[] Sign(byte[] content, byte[] certificateContent) =>
-			winApiCrypt.Sign(content, certificateContent);
+		public byte[] Sign(byte[] content, byte[] certificateContent)
+		{
+			return winApiCrypt.Sign(content, certificateContent);	
+		}
 
-		public List<byte[]> VerifySignature(byte[] content, byte[] signatures) =>
-			winApiCrypt.VerifySignature(content, signatures);
+		public List<byte[]> VerifySignature(byte[] content, byte[] signatures)
+		{
+			return winApiCrypt.VerifySignature(content, signatures);
+		}
 
-		public byte[] Decrypt(byte[] encryptedContent, bool userLocalSystemStorage = false) =>
-			winApiCrypt.Decrypt(encryptedContent, userLocalSystemStorage);
+		public byte[] Decrypt(byte[] encryptedContent, bool userLocalSystemStorage = false)
+		{
+			return winApiCrypt.Decrypt(encryptedContent, userLocalSystemStorage);
+		}
 
-		public List<X509Certificate2> GetPersonalCertificates(bool onlyWithPrivateKey, bool useLocalSystemStorage = false) =>
-			winApiCrypt.GetPersonalCertificates(onlyWithPrivateKey, useLocalSystemStorage);
+		public List<X509Certificate2> GetPersonalCertificates(bool onlyWithPrivateKey, bool useLocalSystemStorage = false)
+		{
+			return winApiCrypt.GetPersonalCertificates(onlyWithPrivateKey, useLocalSystemStorage);	
+		}
 
-		public X509Certificate2 GetCertificateWithPrivateKey(string thumbprint, bool useLocalSystemStorage = false) =>
-			winApiCrypt.GetCertificateWithPrivateKey(thumbprint, useLocalSystemStorage);
+		public X509Certificate2 GetCertificateWithPrivateKey(string thumbprint, bool useLocalSystemStorage = false)
+		{
+			return winApiCrypt.GetCertificateWithPrivateKey(thumbprint, useLocalSystemStorage);
+		}
 
 		public void Dispose()
 		{

--- a/src/Cryptography/StreamCrypt.cs
+++ b/src/Cryptography/StreamCrypt.cs
@@ -118,9 +118,9 @@ namespace Diadoc.Api.Cryptography
 		public byte[] Encode([NotNull] byte[] content)
 		{
 			if (content == null)
-				throw new ArgumentNullException(nameof(content));
+				throw new ArgumentNullException("content");
 			if (content.Length == 0)
-				throw new ArgumentOutOfRangeException(nameof(content), nameof(content) + " is empty");
+				throw new ArgumentOutOfRangeException("content", "[content] is empty");
 
 
 			using (var memoryStream = new MemoryStream())

